### DISCUSSION TITLE
default symfony 7 webapp required mercure with make:entity ?

### DISCUSSION THREAD

**Discussion by stloc** [symfony/symfony#54285] [Q&A] [answered]
when I want to create simple entity I have the error :
```
symfony console make:entity
Class name of the entity to create or update (e.g. AgreeableKangaroo):
 > Test                                                                                                                        
 [ERROR] Please run "composer require symfony/mercure". It is needed to broadcast entities. 
```

How to make without mercure ?

**Comment by stloc** [comment]
I notice this only happens If the argument is missing and the command will ask for the entity class name interactively

**Comment by stof** [comment]
this looks like a bug in https://github.com/symfony/maker-bundle/. It should not complain about symfony/mercure without asking you whether you want to broadcast entities.

**Comment by smnandre** [comment, marked as answer]
@stloc This was a small bug.. but the good news is: it has been [fixed](https://github.com/symfony/maker-bundle/pull/1459) in symfony/maker-bundle:1.5.1 

If you update your version of maker bundle that should be good now !
